connectors/src/connectors/notion/lib/notion_api.ts

```diff
@@ -30,6 +31,14 @@ import mainLogger from "@connectors/logger/logger";
 
 const logger = mainLogger.child({ provider: "notion" });
 
+const notionClientLogger = (
+  level: LogLevel,
+  message: string,
+  extraInfo: Record<string, unknown>
+) => {
+  console.log(`[${level}]: ${message}`, extraInfo);
```

Comment:
Why not directly use a logger here? potentially deactivating after that?

Reply:
We def don't want those logs to clutter our DD errors no ? The'll handle the requests to notion, and we'll do the logging if where we need it wdyt ?

Reply:
I'm not too worried about cluttering. My point is that console.log will clutter more than logger ?

Reply:
I removed the console.log and no-op instead > want me to add back a log in info?

Reply:
OK added back with a logger.info so we keep them but stop being spammed with errors